beamer/config/commands.py

```diff
@@ -133,9 +137,15 @@ def read(
         w3, (request_manager, fill_manager), start_block=start_block, confirmation_blocks=0
     )
     events = fetcher.fetch()
+    events = list(filter(_is_config_update_event, events))
     for event in events:
         _replay_event(w3, deployment, config, event)
 
+    if events:
```

Comment:
nitpick: this could be misleading, I could do updateChain with the exact same values resulting in _is_config_update_event  to be true. Or the values could have been changed and then changed back.
I think it would be more appropriate to check the original state with the state after applying new events and evaluate based on this. Depending on how much effort it is...

Reply:
looking at the log info provided, I think this is actually good enough, so nvm